Aquality.Appium.Mobile/src/Aquality.Appium.Mobile/Applications/LocalApplicationFactory.cs

```diff
@@ -0,0 +1,22 @@
+﻿using OpenQA.Selenium.Appium.Service;
+using OpenQA.Selenium.Appium.Service.Options;
+using System.Collections.Generic;
+
+namespace Aquality.Appium.Mobile.Applications
+{
+    public class LocalApplicationFactory : ApplicationFactory
+    {
+        public override IMobileApplication Application
+        {
+            get
+            {
+                var optionsCollector = new OptionCollector().AddArguments(new KeyValuePair<string, string>("--allow-insecure", "chromedriver_autodownload"));
```

Comment:
should not it be moved into json profile?

Reply:
probably yes, thanks, would be fixed